vector/src/main/java/im/vector/activity/CommonActivityUtils.java

```diff
@@ -1206,10 +1206,10 @@ public void run() {
                                                    Room room = finalSession.getDataHandler().getRoom((String) params.get(VectorRoomActivity.EXTRA_ROOM_ID));
 
                                                    if ((null != room) && room.isInvited()) {
-                                                       String displayName = VectorUtils.getRoomDisplayName(fromActivity, finalSession, room);
+                                                       String roomName = (String) params.get(VectorRoomActivity.EXTRA_DEFAULT_NAME);
```

Comment:
The VectorUtils.getRoomDisplayName call must be restored here
`roomName`is null here according to the check line 1204